build/Build.cs

```diff
@@ -90,6 +91,16 @@ class Build : NukeBuild
                 .SetVerbosity(DotNetVerbosity.Normal)
                 .SetProperty("NuspecFile", "../../build/Octopus.Server.Extensibility.Authentication.Ldap.nuspec")
                 .SetProperty("NuspecProperties", $"Version={OctoVersionInfo.NuGetVersion}"));
+
+            DotNetPack(_ => _
```

Comment:
Are there any special package requirements for client packages?

Reply:
Updated metadata in the csproj based on DirectoryServices extension